File: nnc projects/me.php
<?php
$servername = "localhost";
$username = "username";
$password = "password";
$dbname = "car_sales";

$conn = new mysqli($servername, $username, $password, $dbname);

if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

if ($_SERVER['REQUEST_METHOD'] == 'POST' && $_SERVER['REQUEST_URI'] == '/add_car') {
    $data = json_decode(file_get_contents('php://input'), true);
    $make = $data['make'];
    $model = $data['model'];
    $year = $data['year'];
    $price = $data['price'];

    $stmt = $conn->prepare("INSERT INTO cars (make, model, year, price) VALUES (?, ?, ?, ?)");
    $stmt->bind_param("ssii", $make, $model, $year, $price);
    $stmt->execute();
    $stmt->close();

    echo json_encode(["status" => "success", "id" => $conn->insert_id]);
}

if ($_SERVER['REQUEST_METHOD'] == 'GET' && $_SERVER['REQUEST_URI'] == '/get_cars') {
    $result = $conn->query("SELECT * FROM cars");
    $cars = [];
    while ($row = $result->fetch_assoc()) {
        $cars[] = $row;
    }
    echo json_encode($cars);
}

$conn->close();
?>

File: nnc projects/me.py
from flask import Flask, request, jsonify
import mysql.connector

app = Flask(__name__)

conn = mysql.connector.connect(
    host="localhost",
    user="username",
    password="password",
    database="car_sales"
)

@app.route('/add_car', methods=['POST'])
def add_car():
    data = request.get_json()
    cursor = conn.cursor()
    cursor.execute("INSERT INTO cars (make, model, year, price) VALUES (%s, %s, %s, %s)",
                   (data['make'], data['model'], data['year'], data['price']))
    conn.commit()
    return jsonify({"status": "success", "id": cursor.lastrowid})

@app.route('/get_cars', methods=['GET'])
def get_cars():
    cursor = conn.cursor()
    cursor.execute("SELECT * FROM cars")
    cars = cursor.fetchall()
    return jsonify(cars)

if __name__ == '__main__':
    app.run(debug=True)
